.gitignore

```diff
@@ -1,6 +1,5 @@
 # Dependency directory
 node_modules
-build
```

Comment:
Is there a reason you want to include the built code in the source? It is built before publishing to npm (and build files are included).